src/relentless/simulate/hoomd.py

```diff
@@ -1418,7 +1459,19 @@ def _post_run_v2(self, sim, sim_op):
         del sim[self]["_gsd"]
 
     def process(self, sim, sim_op):
-        pass
+        file_format = analyze.WriteTrajectory._detect_format(self.filename, self.format)
```

Comment:
Same here, maybe we should do the fix above to make it clearer how the `filename` is being used consistently throughout?